UIforETW/DirectoryMonitor.cpp

```diff
@@ -62,21 +61,23 @@ DWORD DirectoryMonitor::DirectoryMonitorThread()
 			return 0;
 
 		default:
-			assert(0);
+			ATLASSERT(0);
 			break;
 		}
 	}
   // Unreachable.
 }
 
+_Pre_satisfies_(this->hThread_ == 0)

```

Comment:
It's too bad these annotations are ugly. I definitely don't want hundreds of annotations because that gets unmaintainable and then you end up with buggy annotations and unreadable code.